app/src/test/java/by/academy/jc/ht/shorokhov/ht10/SpyStorageTest.java

```diff
@@ -0,0 +1,35 @@
+package by.academy.jc.ht.shorokhov.ht10;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class SpyStorageTest {
```

Comment:
Test should provide positive and negative scenario: throw exception and catch it, and then throw and do not catch.